website/content/api-docs/auth/ldap.mdx

```diff
@@ -97,7 +97,7 @@ This endpoint configures the LDAP auth method.
 - `dereference_aliases` `(string: never)` - When aliases should be dereferenced
   on search operations. Accepted values are 'never', 'finding', 'searching',
   'always'. Defaults to 'never'.
-- `max_page_size` `(int: math.MaxInt32)` - If set to a value greater than 0, the LDAP
+- `max_page_size` `(int: -1)` - If set to a value greater than 0, the LDAP
   backend will use the LDAP server's paged search control to request pages of
   up to the given size. This can be used to avoid hitting the LDAP server's
   maximum result size limit. A value of 0 will be interpreted by the LDAP
```

Comment:
> A value of 0 will be interpreted by the LDAP server as unlimited.

Not true. A size of 0 is invalid and causes the LDAP server to return no results.

Reply:
https://github.com/go-ldap/ldap/blob/v2.5.1/search.go#L305

Reply:
(See response above)